analytics/README.md

```diff
@@ -0,0 +1,18 @@
+# TournesolDataViz
+
+Tournesol data visualization tool is a Streamlit app to visualize [tournesol.app](https://tournesol.app/) data.
+
+
+## Run Tournesol data visualization tool
+
+Install the requirement packages:
+
```

Comment:
A suggestion, to make sure that the developers won't mix the back end and Streamlit dependencies. 

```suggestion
## Install

Create a new Python virtual environment, and install the requirements. Be sure to use one virtual environment per Python projects in the Tournesol repository to not mix dependencies.
```